demo/src/client/ToDoListPage/ToDoList.tsx

```diff
@@ -38,7 +38,7 @@ export const ToDoList: FC<ToDoListProps> = memo(({ items, onItemClick }) => {
             <div className="panel-body">
                 <p>To-Do:</p>
 
-                {!hasItems && <p>Add a To-Do to your list.</p>}
+                {!hasItems && <p className={'todo-empty'}>Add a To-Do to your list.</p>}
```

Comment:
```suggestion
                {!hasItems && <p className="todo-empty">Add a To-Do to your list.</p>}
```
If you use the double quotes for className, IntelliJ can actually navigate to the class in the scss file, which is handy.

Reply:
Ah, I didn't realize that. That's very helpful. Done.